packages/ui/src/components/WelcomeSheet.tsx

```diff
@@ -0,0 +1,105 @@
+import { useCallback } from 'react';
+import { Image } from 'react-native';
+
+import { Text, View, XStack, YStack } from '../core';
+import { Icon } from './Icon';
+import { Sheet } from './Sheet';
+
+export function WelcomeSheet({
+  open,
+  onOpenChange,
+}: {
+  open: boolean;
+  onOpenChange: (open: boolean) => void;
+}) {
+  const handleDismiss = useCallback(() => {
+    onOpenChange(false);
+  }, [onOpenChange]);
+
+  return (
+    <Sheet
+      open={open}
+      onOpenChange={handleDismiss}
+      snapPoints={[85]}
```

Comment:
do we need to explicitly set this vs using `fit` mode? Felt like a lot of white space at the bottom on my device

Reply:
Wasn't aware of that! Will fix.